share/models/people.py

```diff
@@ -16,25 +16,17 @@ def __str__(self):
         return self.email
 
 
-class Identifier(ShareObject):
```

Comment:
I'm not certain that combining `Link`s and `Identifier`s is the right thing to do. It makes sense from a simplicity standpoint. I'd be curious to see the performance of  splitting heavy reading/writing between two tables or just one. `Identifier` will be the most active table in the database, it probably is already.